clients/src/test/java/org/apache/kafka/common/config/ConfigDefTest.java

```diff
@@ -171,4 +171,29 @@ private void testValidators(Type type, Validator validator, Object defaultVal, O
             }
         }
     }
+
+    @Test
+    public void toMarkdown() {
+        final String expected = "| Name                                  | Description                                                                                                                                                                                                                                                                                    | Type     | Default                   | Valid Values | Importance |\n" +
```

Comment:
Smaller test case with custom ConfigDef for this test? This is super dependent on config docs used elsewhere in a way that'll make it a real pain to update as any changes are made to that ConfigDef.